src/pages/index.js

```diff
@@ -21,79 +21,187 @@ import {
   addCardModal,
   addCardFormElement,
   initialCards,
+  editAvatarButton,
+  avatarEditForm,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupConfirmation from "../components/PopupConfirmation.js";
 
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addCardFormElement);
+const avatarFormValidator = new FormValidator(config, avatarEditForm);
 
 editFormValidator.enableValidation();
 addFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
-function createCard(item) {
-  const card = new Card(item, "#card-template", handleImageClick);
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "05177711-40d5-4e6a-abe7-ba1645f6efc9",
+    "Content-Type": "application/json",
+  },
+});
+
+function createCard(data) {
+  const card = new Card(
+    data,
+    "#card-template",
+    handleImageClick,
+    handleDeleteCard,
+    handleLikeClick
+  );
   return card.getView();
 }
 
-const cardListEl = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      cardListEl.addItem(createCard(data));
-    },
-  },
-
-  ".cards__list"
-);
+function renderItems(items) {
+  items.forEach((item) => {
+    this._renderer(item);
+  });
+}
+api
+  .getInitialCards()
+  .then((cards) => {
+    const cardSection = new Section(
```

Comment:
Only **one instance** of `Section` should be created in the project because there is only **one** section for the cards.